db/seeds.rb

```diff
@@ -1,8 +1,49 @@
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Company.destroy_all
```

Comment:
Running `bin/rails db:seed` will remove all the companies from the your database. For now maybe it is not that important, but later (when you will have real data) it may be problematic.

You could do:
```ruby
companies_attrs = [
  {name: 'ALIOR', symbol: 'PLALIOR00045'),
  # ...
]

companies.each { |company_attrs| Company.find_or_create_by!(company_attrs) }
```

https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-find_or_create_by-21

Then you would be able to run `bin/rails db:seed` multiple times without adding the duplicated companies.

Reply:
Yes, of course. Anyway this is what I start looking for today, because of adding company from downloaded list :)